packages/design-tokens/stories/test-utils/getCSSVariablesFromStylesheet.ts

```diff
@@ -1,7 +1,10 @@
-import fs from 'fs';
-import path from 'path';
-import postcss from 'postcss';
+/* eslint-disable no-restricted-globals */
+/* eslint-disable id-denylist */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable @typescript-eslint/prefer-for-of */
+/* eslint-disable require-unicode-regexp */
```

Comment:
### Why do we need to get CSS variables from the browser's computed styles?

We need to read CSS variables directly from the browser's computed styles instead of importing from `packages/design-tokens/src/css` because:

1. **Build Processing**: CSS files are processed and transformed during the build process. The raw source files don't reflect the final values that will be used in the browser.

2. **Computed Values**: Some CSS variables may be dynamically computed or inherited from parent elements. We need the browser's computed values to see the actual results.

3. **Theme Resolution**: Theme variations (light/dark) are applied through CSS cascade and need to be computed by the browser to get the correct values. Direct file imports won't show these resolved values.

4. **CSS Transformations**: PostCSS and other build tools may modify the original CSS values. Reading from the browser ensures we see the final, transformed values.

This is why we need to:
- Disable certain ESLint rules to allow browser globals
- Create a temporary div to test theme variations
- Read from `document.styleSheets` instead of importing CSS files directly

The stories in `BrandColors.stories.tsx` and `ThemeColors.stories.tsx` use this approach to accurately display the final, computed values that users will see in the application.